cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/client/AzureClient.java

```diff
@@ -90,13 +97,16 @@
 
     private final Azure azure;
 
+    private final privatednsManager privatednsManager;
```

Comment:
privatednsManager - > PrivateDnsManager

Reply:
This is an Azure internal class, with its own naming.